from0k2bp.tex

```diff
@@ -2143,7 +2143,7 @@
 can see that this requires:
 \begin{align*}
 & \textbf{H}' = \textbf{y}^{-n}\textbf{H} \\
-& P = A + xS -zG + \left(z\textbf{y}^n + z^2\textbf{2}^n\right)\textbf{H}' \\
+& P = A + xS -z\textbf{G} + \left(z\textbf{y}^n + z^2\textbf{2}^n\right)\textbf{H}' \\
```

Comment:
z is committed to via a *single* generator G. Whereas the vector (bolded)G is used as a shorthand. See eqns (2), (3), (4) earlier on.

Reply:
I think you might be confusing the inner product proof (which has `zG`) with the range proof (which has `z\vec{G}`). We need `z\vec{G}` to extract `a_L` and `a_R` (see "Then consider coefficients of G:").